### DISCUSSION TITLE
Migration of customs ParamConverters to remove sensioBundle dep

### DISCUSSION THREAD

**Discussion by etshy** [symfony/symfony#57730] [Q&A] [answered]
We have a few custom ParamConverter that takes values from query/payload and creates automatically the typeHinted object in controller method, without attributes/annotation used.

I saw there is the `MapRequestPayload` now but i don't see anything to make a custom "mapper" like we had with paramConverter.

Should I use [`value resolvers`](https://symfony.com/doc/current/controller/value_resolver.html#adding-a-custom-value-resolver) or is there another proper alternative ? 
Itlooks a bit different but could do the same job.

**Comment by derrabus** [comment, marked as answer]
> Should I use [`value resolvers`](https://symfony.com/doc/current/controller/value_resolver.html#adding-a-custom-value-resolver)

Yes. This is the replacement.

**Reply by etshy** [reply]
Thanks for confirmation !
